gremlin-client/src/conversion.rs

```diff
@@ -117,6 +119,7 @@ impl_from_gvalue!(f32, GValue::Float);
 impl_from_gvalue!(f64, GValue::Double);
 impl_from_gvalue!(i32, GValue::Int32);
 impl_from_gvalue!(i64, GValue::Int64);
+impl_from_gvalue!(bool, GValue::Bool);
```

Comment:
Not strictly related to cardinality but its omission was causing missing trait compiler errors, hope it's okay I added it.

Reply:
:+1: